atd-etl/cris_import/cris_import.py

```diff
@@ -81,7 +82,21 @@ def main():
     SFTP_ENDPOINT_SSH_PRIVATE_KEY = secrets["sftp_endpoint_private_key"]
 
     # 🥩 & 🥔
-    zip_location = download_extract_archives()
+    #zip_location = download_extract_archives()
+
+    local_mode = False
+    if bool(glob.glob('/app/development_extracts/*.zip')):
+        local_mode = True
+
+    zip_location = None
+    if not local_mode: # Production
+        zip_location = download_archives()
+    else: # Development. Put a zip in the development_extracts directory to use it.
+        zip_location = specify_extract_location()
+
+    if not zip_location:
+        return
+
```

Comment:
Backport this functionality from LDM, it's super helpful for testing this sort of thing.

Reply:
Cool to see this getting added here. I could see a tiny blurb about this in the readme being helpful.

Reply:
Yea, totally good call about adding a blurb. I put one in the readme in ee872c02d62e43329747fd7fa00d32bd16564d06.